File: Golang/1008.go
package main

import (
	"fmt"
)

func main() {

	var id_funcionario int
	var horas_trabalhadas, valor_hora float64

	fmt.Scan(&id_funcionario)
	fmt.Scan(&horas_trabalhadas)
	fmt.Scan(&valor_hora)

	salario := horas_trabalhadas * valor_hora

	fmt.Printf("NUMBER = %d\n", id_funcionario)
	fmt.Printf("SALARY = U$ %.2f\n", salario)
}

File: Elixir/1008.exs
id_funcionario = IO.gets("") |> String.trim() |> String.to_integer
horas_trabalhadas = IO.gets("") |> String.trim() |> String.to_integer
valor_hora = IO.gets("") |> String.trim() |> String.to_float

salario = (valor_hora * horas_trabalhadas)

IO.puts "NUMBER = #{id_funcionario}"
IO.puts "SALARY = U$ #{:erlang.float_to_binary(salario, decimals: 2)}"
